runtime/parachains/src/disputes.rs

```diff
@@ -1026,6 +1056,37 @@ impl<T: Config> Pallet<T> {
 			// However, 3 sequential calls, where the first increments,
 			// the second decrements, and the third increments would be allowed.
 			SpamSlots::<T>::insert(&set.session, spam_slots);
+
+			// This is only relevant in cases where it's the first vote and the state
+			// would hence hold a onesided dispute. If a onesided dispute can never be
+			// started, by induction, we can never enter a state of a one sided dispute.
+			if spam_filter_struck && first_votes {
+				let mut vote_for_count = 0_u64;
+				let mut vote_against_count = 0_u64;
+				// Since this is the first set of statements for the dispute,
+				// it's sufficient to count the votes in the statement set after they
+				set.statements.iter().for_each(|(statement, v_i)| {
+					match statement {
+						// `summary.new_flags` contains the spam free votes.
+						// Note that this does not distinguish between pro or cons votes,
+						// since allowing both of them, even if the spam threshold would be reached
+						// is a good thing.
+						// Overflow is no concern, disputes are limited by weight.
+						DisputeStatement::Valid(_) => {
+							if Some(true) == summary.new_flags.get(v_i) {
+							vote_for_count += 1;
+						}}
+						DisputeStatement::Invalid(_) => {
+							if Some(true) == summary.new_flags.get(v_i) {
```

Comment:
@eskimor it's used here (and nowhere else) to make sure we only count newly added votes, which wasn't done before